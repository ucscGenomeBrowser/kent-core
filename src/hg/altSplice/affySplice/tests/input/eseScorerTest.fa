>G6820594@J909190_RC@j_at
gttcttccagttcttttaacatgtcaaactctggagatttgttcatgagcgtgcagtcactcaatggggattcttaccaaggggcccaggttggagccaacgtgcaatcacag
>G6813886@J909757@j_at
cagtgggagacatttagtgaacgctcttcaagctcacaaactctgacccaatttgattctaacattgcaccagctgatcct

