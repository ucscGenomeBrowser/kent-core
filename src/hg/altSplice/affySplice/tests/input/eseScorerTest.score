motif: sf2.asf 1.956
A 	-1.14 	0.62 	-1.58 	1.32 	-1.58 	-1.58 	0.62
C 	1.37 	-1.1 	0.73 	0.33 	0.94 	-1.58 	-1.58
G 	-0.21 	0.17 	0.48 	-1.58 	0.33 	0.99 	-0.11
T 	-1.58 	-0.5 	-1.58 	-1.13 	-1.58 	-1.13 	0.27

motif: sc35 2.383
A 	-0.88 	0.09 	-0.06 	-1.58 	0.09 	-0.41 	-0.06 	0.23
C 	-1.16 	-1.58 	0.95 	1.11 	0.56 	0.86 	0.32 	-1.58
G 	0.87 	0.45 	-1.36 	-1.58 	-0.33 	-0.05 	-1.36 	0.68
T 	-1.18 	-0.2 	0.38 	0.88 	-0.2 	-0.86 	0.96 	-1.58

motif: srp40 2.670
A 	-0.13 	-1.58 	1.28 	-0.33 	0.97 	-0.13 	-1.58
C 	0.56 	0.68 	-1.12 	1.24 	-0.77 	0.13 	-0.05
G 	-1.58 	-0.14 	-1.33 	-0.48 	-1.58 	0.44 	0.8
T 	0.92 	0.37 	0.23 	-1.14 	0.72 	-1.58 	-1.58

motif: srp55 2.676
A 	-0.66 	0.11 	-0.66 	0.11 	-1.58 	0.61
C 	0.39 	-1.58 	1.48 	-1.58 	-1.58 	0.98
G 	-1.58 	0.72 	-1.58 	0.72 	0.21 	-0.79
T 	1.22 	-1.58 	-0.07 	-1.58 	1.02 	-1.58
