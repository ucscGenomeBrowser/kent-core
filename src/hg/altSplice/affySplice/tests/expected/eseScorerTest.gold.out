size	sf2.asf	sc35	srp40	srp55
G6820594@J909190_RC@j_at	113	3	5	3	1
G6813886@J909757@j_at	81	3	1	2	1
